jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java

```diff
@@ -175,11 +177,25 @@ protected void addProcessIdFilter() {
                                     f -> addSearchFilter(f, equalsTo(COLUMN_PROCESS_ID, f.getValue())));
     }
 
+    protected void addContainerIdFilter() {
```

Comment:
@bxf12315 here, the same, I think a better name is addDeploymentIdFilter, to align with the kind of information is related to.

Reply:
ok